bentoml/db.py

```
@@ -33,9 +34,19 @@
 
 def init_db(db_url):
     # Use default config if not provided
-    engine = create_engine(
-        db_url, echo=False, connect_args={'check_same_thread': False}
-    )
+    # we have to parse the db url. Depends on what type of it,
+    if is_sqlite_db(db_url):
+        engine = create_engine(
+            db_url, echo=False, connect_args={'check_same_thread': False}
+        )
+    elif is_postgres_db(db_url):
+        if not is_postgres_db_name_exists(db_url):
```

change to "let it fail" and catch the exception as discuessed